Makefile

```diff
@@ -49,7 +49,7 @@ chroma-dependencies: chroma-agent chroma-manager chroma-diagnostics
 chroma-bundles: chroma-dependencies
 
 destroy_cluster: Vagrantfile
-	vagrant destroy
+	vagrant destroy -f
```

Comment:
as you say the abstraction is not very complete, it is probably just implementation detail that the call does not require confirmation on libvirt